pymc/logprob/utils.py

```diff
@@ -320,7 +320,7 @@ def find_negated_var(var):
     return None
 
 
-def get_related_valued_nodes(node: Apply, fgraph: FunctionGraph) -> list[Apply]:
+def get_related_valued_nodes(fgraph: FunctionGraph, node: Apply) -> list[Apply]:
```

Comment:
This is much more natural order used in all sorts of pytensor utilities that require a node/variable and its' fgraph